snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsTypes/CS/keywordsTypes.cs

```diff
@@ -190,15 +190,15 @@ static void Main()
     class DelegateTest
     {
         // Regular method that matches signature:
-        static double Double(double input)
+        static double Foo(double input)
```

Comment:
The original name matched the functionality of the method better, although I can see the confusion from using a method name that matches a numeric type name.

I think we should pick a different name. How about `DoubleAmount` or `TwiceTheValue`. If you have a different idea, that would be great as well.

Reply:
Well, let's use `DoubleAmount` instead of `Foo` as the method name